.circleci/config.yml

```diff
@@ -286,6 +286,47 @@ jobs:
                docker push $DOCKERHUB_REPOSITORY/$DOCKER_IMAGE:latest
              fi
 
+  gh-pages-deploy:
+    machine:
+        image: ubuntu-1604:201903-01
+    steps:
+      - checkout
+      - run:
+          name: Install elixir
+          command: |
+              wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb && \
+              sudo dpkg -i erlang-solutions_1.0_all.deb && \
+              sudo apt-get update && \
+              sudo apt-get install -y esl-erlang=1:21.0 && \
+              sudo apt-get install -y elixir=1.8.0-1
```

Comment:
Does it make a difference in the UI or structure of the generated files if newer Elixir version is used?

Reply:
Some dependencies seem to break with Elixir 1.8, and nevertheless these are the versions used in integration tests, so I think we could consider upgrading versions as part of a different ticket 🙂